lochness/redcap/__init__.py

```diff
@@ -52,44 +55,49 @@ def initialize_metadata(Lochness: 'Lochness object',
                         study_name: str,
                         redcap_id_colname: str,
                         redcap_consent_colname: str,
-                        multistudy: bool = True) -> None:
-    '''Initialize metadata.csv by pulling data from REDCap
+                        multistudy: bool = True,
+                        upenn: bool = False) -> None:
+    '''Initialize metadata.csv by pulling data from REDCap for Pronet project
 
     Key arguments:
         Lochness: Lochness object
-        study_name: Name of the study, str.
+        study_name: Name of the study, str. eg) PronetLA
         redcap_id_colname: Name of the ID field name in REDCap, str.
         redcap_consent_colname: Name of the consent date field name in REDCap,
                                 str.
-        multistudy: True if the redcap repo contains more than one study's data
+        multistudy: True if the redcap repo contains multisite data, bool.
+        upenn: True if upenn redcap is included in the source list, bool.
     '''
-    study_redcap = Lochness['keyring'][f'redcap.{study_name}']
-    api_url = study_redcap['URL'] + '/api/'
-    api_key = study_redcap['API_TOKEN'][study_name]
+    if multistudy:
+        # specific to DPACC project
+        site_two_letters_study = study_name[-2:]  # 'LA'
```

Comment:
This is a long argument name. There are four places where it is used. It should just be abridged as `site_code`.